templates/blog-post.html

```diff
@@ -27,5 +40,30 @@ <h2 class="mb-0">{{ this.title }}</h2>
       </p>
     </div>
   </div>
+{% if this.series|length %}
+<table class="table table-sm table-striped mt-3" id="series">
+<thead class="thead-dark">
+<tr>
+<th>
+All posts under the Series "<span style="font-size:1.15rem">{{ this.series }}<span>"
+</th>
+</tr>
+</thead>
+<tbody>
+{% for post in series.children %}
+      {% set author = post.parent.parent.children.get('authors').children.get(post.author) %}
+      <tr>
+        <td>
+        <a href="{{ post|url }}">{{ post.title }}</a>
```

Comment:
I think just the title and URL is enough, we don't need the authors name and date here.